src/state/QuestWrapperProvider.tsx

```diff
@@ -0,0 +1,75 @@
+import { createContext, useContext, ReactNode } from 'react'
+import { TFunction } from 'i18next'
+import { TrackEventFn } from '@/types/analytics'
+import {
+  ConfirmClaimParams,
+  DepositContract,
+  Quest,
+  Reward,
+  RewardClaimSignature,
+  Runner
+} from '@hyperplay/utils'
+
+interface QuestWrapperContextValue {
```

Comment:
Could we move this to `types/quests`? That way functions like your `useGetRewards` hook could import it and use its types more easily